charts/gardener/provider-local/extension/doc.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//go:generate ../../../../hack/generate-controller-registration.sh provider-local . ../../../../VERSION ../../../../example/provider-local/base/controller-registration.yaml DNSProvider:local DNSRecord:local ControlPlane:local Infrastructure:local Network:local OperatingSystemConfig:local Worker:local
+//go:generate ../../../../hack/generate-controller-registration.sh provider-local . v0.0.0 ../../../../example/provider-local/base/controller-registration.yaml DNSProvider:local DNSRecord:local ControlPlane:local Infrastructure:local Network:local OperatingSystemConfig:local Worker:local
```

Comment:
If I would now like to pass a version that is defined in the `VERSION` file, can I simply do `$(cat VERSION)`? I expect, that this doesn't work with `go:generate`

Reply:
You can't, you would have to use `sh -c "..."` instead.

Reply:
Hmm, ok. Fine then